src/config.cr

```diff
@@ -362,11 +362,9 @@ module Wsman
       )
     end
 
-    def env_file_custom(site_name)
-      File.join(
-        @config.docker_environment_dir,
-        "#{@config.docker_environment_prefix}#{site_name}-custom"
-      )
+    # All -<postfix> postfixed envfile
+    def env_files_custom(site_name)
+      env_files = Dir["#{@config.docker_environment_dir}/#{@config.docker_environment_prefix}#{site_name}-*"]
```

Comment:
The `env_files` variable looks unused here, this could be replaced with:

```crystal
def env_files_custom(site_name)
  Dir["#{@config.docker_environment_dir}/#{@config.docker_environment_prefix}#{site_name}-*"]
end
```